health/service/src/main/java/com/cerner/beadledom/health/HealthDependency.java

```diff
@@ -35,10 +50,18 @@
   /**
    * Returns a Boolean to indicate if this check is a primary check for this service.
```

Comment:
Added the deprecated tag as well as told them to use `{@link #isPrimary()}` to access and added a sentence about using the constructors instead of overriding the method: https://github.com/cerner/beadledom/pull/123/commits/95686b23b05f48a497bd6fac0a15acfb9712165b